Makefile

```diff
@@ -145,14 +145,20 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Path used to import Gatekeeper manifests
-IMPORT_MANIFESTS_PATH ?= https://github.com/open-policy-agent/gatekeeper
+IMPORT_MANIFESTS_PATH ?= https://github.com/stolostron/gatekeeper
 TMP_IMPORT_MANIFESTS_PATH := $(shell mktemp -d)
 GATEKEEPER_MANIFEST_DIR ?= config/gatekeeper
 
+GATEKEEPER_TAG = $(shell curl -L https://api.github.com/repos/stolostron/gatekeeper/tags | jq -r '.[].name' | sort --version-sort | grep $(shell echo $(GATEKEEPER_VERSION) | cut -d '.' -f 1-3) | tail -1 )
```

Comment:
@dhaiducek do you know if we'll run into GitHub API rate limiting with this? It seems we would use an API request every time we executed the Makefile.

Reply:
Also, I really don't want an HTTP request to be made every time I use the Makefile.

Reply:
I'm not sure the variable is expanded every time since it's not referenced except in `import-manifests` (though I'm never 100% sure with Make... 🤷‍♂️ ) . But to your point, we could probably just use `git tag` and clone/fetch the repo tags instead of a targeted clone. Looks like we might be able to do something like (untested) this?
```
git clone --no-checkout ...
git fetch --tags
git checkout $(shell git tag -l "$(shell $(GATEKEEPER_VERSION) | cut -d '.' -f 1-2)*" | sort --version-sort | tail -1)
```

Reply:
@mprahl Why calling HTTP request is bad?

Reply:
@dhaiducek since the variable is not defined in a target, I think it would be expanded every time you call Make.

Reply:
Actually, it does seem like if the variable is not referenced it doesn't call the shell command.

Reply:
> @mprahl Why calling HTTP request is bad?

It's not bad if it's required but when it can be avoided, it reduces the chance of failure. For example, if I'm developing without an internet connection, the GitHub API is down, GitHub API rate limiting, or the network connection is flaky.

Since I misunderstood when the shell command actually gets run, I think it's not as bad as I thought since I think it's only called under `import-manifests`.

Reply:
@dhaiducek  good to go?

Reply:
<img width="1271" alt="Screenshot 2024-04-23 at 7 55 50 AM" src="https://github.com/stolostron/gatekeeper-operator/assets/121250524/fb82d8be-9932-4e55-9c3c-7e05daaf1f90">
These are our tag I thought our tags should be like these

Reply:
Right, but if the version tag referenced in the operator is v3.15.1, for example, but we've updated Gatekeeper to have a v3.15.2, we want v3.15.2 to be picked up here by grepping for v3.15 and for `import-manifests` to then fail when it picks up the correct v3.15.2 tag.